src/h5web/visualizations/line/config.ts

```diff
@@ -1,13 +1,19 @@
 import { StorageConfig, createPersistableState } from '../../storage-utils';
 
-type LineVisConfig = {};
+type LineVisConfig = {
+  showGrid: boolean;
+  toggleGrid: () => void;
+};
 
 const STORAGE_CONFIG: StorageConfig = {
-  storageId: 'h5web:line',
-  itemsToPersist: [],
+  storageId: 'h5web:linevis',
```

Comment:
Would naming the components `LinePlot<...>` instead of `LineVis` be better? Just thinking that we don't use `HeatmapVis<...>`, and "plot" would make more sense for, say, the future scatter plot visualization.

Reply:
I used `Line...` to be consistent with the naming of `Heatmap` components